interfaces/builtin/usb_raw.go

```diff
@@ -0,0 +1,45 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const usbrawConnectedPlugAppArmor = `
+# Description: Allow raw access to all connected USB devices.
+# Reserved because this gives privileged access to the system.
+# Usage: reserved
+/dev/bus/usb/[0-9][0-9][0-9]/[0-9][0-9][0-9] rw,
+
+# Allow detection of usb devices. Leaks plugged in USB device info
+/sys/bus/usb/devices/ r,
+/sys/devices/pci**/usb[0-9]**/idVendor r,
+￼/sys/devices/pci**/usb[0-9]**/idProduct r,
+`
```

Comment:
it seems that many other files are needed, this are the denials from usb-devices:
```
matteo@matteo-desktop:~/src/canonical/snapd$ usb-utils.usb-devices &>/dev/null
matteo@matteo-desktop:~/src/canonical/snapd$ dmesg |awk -F'"' '/apparmor="DENIED"/{print$8}'
/sys/devices/pci0000:00/0000:00:14.0/usb1/maxchild
/sys/devices/pci0000:00/0000:00:14.0/usb1/version
/sys/devices/pci0000:00/0000:00:14.0/usb1/bDeviceClass
/sys/devices/pci0000:00/0000:00:14.0/usb1/bDeviceSubClass
/sys/devices/pci0000:00/0000:00:14.0/usb1/bDeviceProtocol
/sys/devices/pci0000:00/0000:00:14.0/usb1/bMaxPacketSize0
/sys/devices/pci0000:00/0000:00:14.0/usb1/bNumConfigurations
/sys/devices/pci0000:00/0000:00:14.0/usb1/bcdDevice
/sys/devices/pci0000:00/0000:00:14.0/usb1/bcdDevice
/sys/devices/pci0000:00/0000:00:14.0/usb1/manufacturer
/sys/devices/pci0000:00/0000:00:14.0/usb1/speed
```
what about just allowing `/sys/devices/pci**/usb[0-9]** r,`?